### PR TITLE
Improve get resources

### PR DESCRIPTION
**Pull request by lborguetti**
Guys, 

this MR added functions to get id of vnet, route tables and routes also added a function to update route in a route table.

```sh
# azure_vnet_get_id will return the Virtual Network ID.
# `name` is the Virtual Network name.
# `group` is name of resource group.
fn azure_vnet_get_id(name, group) 
```
```sh
# azure_route_table_get_id will return the Route Table ID.
# `name` is the route table name.
# `group` is name of resource group.
fn azure_route_table_get_id(name, group)
```
```sh
# azure_route_table_route_get_id will return the route ID.
# `name` is the route name in a route table.
# `group` is name of resource group.
# `routetable` is name of route table.
fn azure_route_table_get_id(name, group, routetable)
```

```sh
# fn azure_route_table_route_update creates a route in a route table.
# `instance` is the name of the route instance.
 fn azure_route_table_route_create(instance)
```

We can check when there is a route into routing table and update it if necessary.

### CODE REVIEW

File: azure/route.sh [hunk 1]
```diff
@@ -12,6 +12,33 @@ fn azure_route_table_delete(name, group) {
 	)
 }
 
+# azure_route_table_get_id will return the Route Table ID.
+# `name` is the route table name.
+# `group` is name of resource group.
+fn azure_route_table_get_id(name, group, routetable) {
```
Comment (vitorarins) [inline azure/route.sh]: I think the third parameter is not being used.

File: azure/route.sh [hunk 2]
```diff
@@ -62,3 +101,28 @@ fn azure_route_table_delete_route(name, group, routetable) {
 						--route-table-name $routetable
 	)
 }
+
+# azure_route_table_route_get_id will return the route ID.
+# `name` is the route name in a route table.
+# `group` is name of resource group.
+# `routetable` is name of route table.
+fn azure_route_table_get_id(name, group, routetable) {
```
Comment (vitorarins) [inline azure/route.sh]: Ctrl+C,Ctrl+V detected. I think what you mean is `azure_route_table_route_get_id`

Comment (lborguetti) [inline azure/route.sh]: :man_facepalming:

File: azure/route.sh (hunk 1 above)
Comment (lborguetti) [inline azure/route.sh]: fixed!

File: tests/azure/testdata/add_internet_route_to_route_table.sh [hunk 3]
```diff
@@ -10,6 +10,22 @@ hoptype    = $ARGS[5]
...
+
+route_id <= get_route_id()
+
+if $route_id == "" {
+	route <= azure_route_table_route_new($name, $resgroup, $routetable, $address, $hoptype)
+	
+	azure_route_table_route_create($route)
+}
+
+route_id <= get_route_id()
```
Comment (vitorarins) [inline tests/azure/testdata/add_internet_route_to_route_table.sh]: typo, `get_route_table_id()`

**Comment by vitorarins** [comment]
Nice! :+1: It would be nice to have more opinions on this @tiago4orion @katcipis .

**Comment by lborguetti** [comment]
Tests Ok \o/

**Pull request merged**